### PR TITLE
Integration test for `run_dicom_archive_loader.py`

### PR DESCRIPTION
**Pull request by maximemulder**
This PR mounts the S3 bucket in CI and adds an integration test for `run_dicom_archive_loader.py` !

### CODE REVIEW

**Comment by maximemulder** [comment]
Still kind of barebones but better than nothing for now IMO.

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ env:
   DATABASE_NAME: TestDatabase
   DATABASE_USERNAME: TestUsername
   DATABASE_PASSWORD: TestPassword
+  BUCKET_URL: https://ace-minio-1.loris.ca:9000
+  BUCKET_NAME: loris-rb-data
+  BUCKET_ACCESS_KEY: lorisadmin-ro
+  BUCKET_SECRET_KEY: Tn=qP3LupmXnMuc
```
Comment (cmadjar) [inline .github/workflows/integration.yml:17]: @maximemulder could this be secrets variables? I know this is for read-only data but still, not ideal to get credentials in the code.

**Review by cmadjar — DISMISSED** [review]

File: .github/workflows/integration.yml (hunk 1 above)
Comment (maximemulder) [reply .github/workflows/integration.yml]: According to this doc:
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-encrypted-secrets-in-a-workflow
> With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.
So I guess our options are either:
- Keep the variables secret, but the integration tests must be triggered manually.
- Make the variables public.
Still looking into the above options, I may edit this message in the future.

**Comment by maximemulder** [comment]
There is now a test to check that a file has indeed been inserted in the database !

**Comment by maximemulder** [comment]
Wait, this is not Samir's brain, someone lied to me !!!

image

File: .github/workflows/integration.yml (hunk 1 above)
Comment (maximemulder) [reply .github/workflows/integration.yml:21]: After the setting up of a bucket without keys taking too long to set up with IT, it was decided to hard-code the keys for now.

**Review by laemtl — APPROVED** [review]

**Pull request merged**